core/app/models/spree/reimbursement.rb

```diff
@@ -51,10 +51,12 @@ class IncompleteReimbursementError < StandardError; end
     # These are called if the call to "reimburse!" succeeds.
     class_attribute :reimbursement_success_hooks
     self.reimbursement_success_hooks = []
+    deprecate :reimbursement_success_hooks, 'Add a processor for the :reimbursement_processed event instead.', deprecator: Spree::Deprecation
```

Comment:
Deprecating implies that this will be removed in a future version, but as far as I can tell the changes later in this file have made this entirely non-functional.